File: LuckFour.c
#include<stdio.h>

int main()
{
    int i,test,count;
    char str[20];
    scanf("%d",&test);
    while(--test>=0)
    {
        count=0;
        scanf("%s",&str);
        i=0;
        while(str[i]!='\0')
        {
            if(str[i]=='4')
                count++;
                i++;
        }
        printf("\n%d",count);
    }
    return 0;
}

File: LuckFour.java
import java.util.*;

class LuckFour
{
	public static void main(String[] ar)
	{
		Scanner cin=new Scanner(System.in);
		int te=cin.nextInt();
		cin.nextLine();
		for(int test=0;test<te;test++)
		{
			int count=0,i=0;
			String asd="";
			asd=cin.nextLine();
			char[] ch=asd.toCharArray();
			for(i=0;i<ch.length;i++)
				if(ch[i]=='4')
					count++;
			System.out.println(count);
		}
	}
}